data/packages/com.wayn-group.blob.yml

```diff
@@ -0,0 +1,26 @@
+name: com.wayn-group.blob
+displayName: WAYN Blob
+description: >-
+Repository of additional unity BlobAsset
+
+Known issues
+To work with the unity Entities 17 package, some changes must be done to you local entity package. more details here : https://forum.unity.com/threads/blobarray-with-valuetype-generic-parameters-fails.1045639/ That modification is included in that répository : https://github.com/WAYNGROUP/wayn.blob/blob/ab29fe7da460dbe6b231d959abebc3a76e1704b0/Packages/com.unity.entities%400.17.0-preview.41/Unity.Entities.CodeGen/CecilExtensionMethods.cs#L94-L97
+
```

Comment:
The syntax is invalid. Please check with https://yamlchecker.com/

Add leading spaces to the multiple lines text shall make it work.
```
description: >-
  Repository of additional unity BlobAsset

  Known issues
  To work with the unity Entities 17 package, some changes must be done to you local entity package. more details here : https://forum.unity.com/threads/blobarray-with-valuetype-generic-parameters-fails.1045639/ That modification is included in that répository : https://github.com/WAYNGROUP/wayn.blob/blob/ab29fe7da460dbe6b231d959abebc3a76e1704b0/Packages/com.unity.entities%400.17.0-preview.41/Unity.Entities.CodeGen/CecilExtensionMethods.cs#L94-L97
```

BTW, it may be better to just remove the lines starts with "known issues", to keep the description simple. And add more information to the README file.

Reply:
Changes were applied to correct the syntax.
The know issues part has been removed as it is also mentionned in the README.md.